app/views/layouts/application.html.erb

```diff
@@ -11,10 +11,8 @@
 
   <body>
     <% flash.each do |type, msg| %>
-      <div class="<%= flash_class(type) %>">
-        <%= msg %>
-      </div>
-    <% end %>
-    <%= yield %>
```

Comment:
Any special reason for removing `yield` from here?

Reply:
Without this, rails is not able to render the other views in the application layout